### PR TITLE
Feature/gh 1082 keycard enable 2fa

### LINKED ISSUE #1082: Keycard Enable 2FA Flow
- Intro change text, image
- reuse the flow from Setup2FA
- card Intro is different design
- review text changed
- success screen change image, text

### LINKED ISSUE #1083: Keycard Disable 2FA Flow
- intro changed
- review text changed
- success image, text

### PR DESCRIPTION
**Pull request by sche**
A lot of renamings and adoption of Connect/Disconnect/Replace 2FA
closes #1082 
closes #1083 
coses #1084

### CODE REVIEW

File: MultisigWallet/MultisigWalletApplication/WalletApplicationService.swift [hunk 1]
```diff
@@ -693,7 +697,7 @@ public class WalletApplicationService: Assertable {
             _ = try requestTransactionConfirmationIfNeeded(id)
             tx = DomainRegistry.transactionRepository.find(id: TransactionID(id))!
         }
-        if tx.type == .replaceBrowserExtension || tx.type == .disconnectBrowserExtension {
+        if tx.type == .replaceTwoFAWithAuthenticator || tx.type == .disconnectAuthenticator {
```
Comment (DmitryBespalov) [inline MultisigWallet/MultisigWalletApplication/WalletApplicationService.swift]: do we need here `replaceTwoFAWithStatusKeycard`?

File: MultisigWallet/MultisigWalletDomainModel/Wallet/BrowserExtension/ReplaceTwoFADomainService.swift [hunk 2]
```diff
@@ -250,31 +263,48 @@ open class ReplaceBrowserExtensionDomainService: Assertable {
...
-    func removeOldOwner(from wallet: Wallet) throws {
-        guard let oldOwner = wallet.owner(role: .browserExtension) else { return }
-        try DomainRegistry.communicationService.deletePair(walletID: wallet.id, other: oldOwner.address.value)
+    func removeOldTwoFAOwner(from wallet: Wallet) throws {
+        let owner = wallet.owner(role: .browserExtension) ?? wallet.owner(role: .keycard)
+        guard let oldOwner = owner else { return }
+        if oldOwner.role == .browserExtension {
+            try DomainRegistry.communicationService.deletePair(walletID: wallet.id, other: oldOwner.address.value)
+        }
         wallet.removeOwner(role: oldOwner.role)
```
Comment (DmitryBespalov) [inline MultisigWallet/MultisigWalletDomainModel/Wallet/BrowserExtension/ReplaceTwoFADomainService.swift:302]: do we need to save the wallet here?

File: SafeUIKit/SafeAppUI/en.lproj/Localizable.strings [hunk 3]
```diff
@@ -176,31 +188,31 @@ Please have your recovery phrase ready to disable 2FA.";
 "error" = "Error";
 
 /* Signature invalid */
-"error_keycard_invalid_signature" = "The signature of the keycard is invalid. Please try again.";
+"error_keycard_invalid_signature" = "error_keycard_invalid_signature";
```
Comment (DmitryBespalov) [inline SafeUIKit/SafeAppUI/en.lproj/Localizable.strings]: Translations are gone

**Review by DmitryBespalov — CHANGES_REQUESTED** [review]

File: MultisigWallet/MultisigWalletApplication/WalletApplicationService.swift (hunk 1 above)
Comment (sche) [reply MultisigWallet/MultisigWalletApplication/WalletApplicationService.swift]: I don't know. This was just a renaming.

File: MultisigWallet/MultisigWalletDomainModel/Wallet/BrowserExtension/ReplaceTwoFADomainService.swift (hunk 2 above)
Comment (sche) [reply MultisigWallet/MultisigWalletDomainModel/Wallet/BrowserExtension/ReplaceTwoFADomainService.swift:302]: add(newOwner:) will do it later

File: SafeUIKit/SafeAppUI/en.lproj/Localizable.strings (hunk 3 above)
Comment (sche) [reply SafeUIKit/SafeAppUI/en.lproj/Localizable.strings]: just added missing translations

**Review by DmitryBespalov — APPROVED** [review]

**Pull request merged**